src/app/trainee/revalidation-table/revalidation-table.component.html

```diff
@@ -18,25 +13,13 @@
       <th mat-header-cell scope="col" *matHeaderCellDef>Outcome</th>
       <td mat-cell *matCellDef="let element">
         <div class="outcome" [ngSwitch]="element.gmcOutcome">
-          <mat-icon
-            *ngSwitchCase="revalidationGmcOutcome.APPROVED"
-            aria-hidden="true"
-            class="icon-green"
-          >
+          <mat-icon *ngSwitchCase="'APPROVED'" aria-hidden="true" class="icon-green">
```

Comment:
same here, can the approved, rejected etc strings be obtained from an enum variable?

Reply:
done